examples/02_decoding/plot_miyawaki_reconstruction.py

```diff
@@ -16,66 +16,73 @@
 The code is a bit elaborate as the example uses, as the original article,
 a multiscale prediction on the images seen by the subject.
 
-See also
-:ref:`sphx_glr_auto_examples_02_decoding_plot_miyawaki_encoding.py` for a
-encoding approach for the same dataset.
+For an encoding approach for the same dataset, see also
+:ref:`sphx_glr_auto_examples_02_decoding_plot_miyawaki_encoding.py`
 
 .. include:: ../../../examples/masker_note.rst
 
 """
 
+
+import sys
```

Comment:
Should be below comment. But I don't think this comment is useful at all, so we might be better removing the comment instead...

Reply:
did that